cmd/kubelet/app/server.go

```diff
@@ -651,6 +653,7 @@ type KubeletConfig struct {
 	Auth                           server.AuthInterface
 	Builder                        KubeletBuilder
 	CAdvisorInterface              cadvisor.Interface
+	CalculateVolumeDiskUsageFrequency time.Duration
```

Comment:
Per previous comment - we use frequency to mean period (periods have a duration)